src/js/components/AppListComponent.jsx

```diff
@@ -196,6 +196,18 @@ var AppListComponent = React.createClass({
       .sortBy((app) => {
         return app[sortKey];
       }, state.sortDescending)
+
```

Comment:
OK, will remove the extra lines. You're right, it's uncommon in our code.
Comment inside the sort doesn't makes sense, because I want to comment the whole sort-function and not the first if-block inside the function.